pyproject.toml

```diff
@@ -543,7 +543,8 @@ markers = [
     "integration: mark test as an integration test.",
     "slow: mark tests taking longer than 1 second.",
     "unit: mark a test as a unit test.",
-    "clickhouse: mark a test as Clickhouse-dependent.", 
+    "aws: test uses an AWS service (sqs, sns, s3 etc).",
```

Comment:
@billdirks 
created an `aws` marker.

LMK if you've already created one

Reply:
I've created on also in one of my PRs that has yet to merge: https://github.com/great-expectations/great_expectations/pull/8299/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711R537

Reply:
I think we are trying to keep these alphabetized.

Reply:
Oh also I see you have the marker description as 

>mark all tests that require aws credentials

But I was treating mine as a more general `test relates ot an AWS service (sqs, sns, s3 etc).`.

Reply:
Per our discussion, @billdirks is going to define this marker ad `aws_creds` and I'm removing the marker from this PR.